frontend/src/components/ViewsList.jsx

```diff
@@ -20,12 +20,12 @@ export const ViewsList = ({ templates, cardClickCallback }) => {
         if (cat in templates && templates[cat].length > 0) {
           return (
             <div key={cat} className="mb-8 p-4  ">
-              <h2 className="border-b border-gray-900 pb-2 text-2xl font-bold dark:border-gray-300">
+              <h2 className="border-b border-gray-900 pb-2 text-2xl  font-bold dark:border-gray-300">
```

Comment:
```suggestion
              <h2 className="border-b border-gray-900 pb-2 text-2xl font-bold dark:border-gray-300">
```

Make sure that your prettier (+ Tailwind extension) is properly set up. This should handle those cases already.